03-nlp/02-CLU/clu.http

```diff
@@ -0,0 +1,23 @@
+### CLU
+POST {{ language endpoint }}/language/:analyze-conversations?api-version=2022-10-01-preview 
+Content-Type: application/json
+Ocp-Apim-Subscription-Key: {{ Language-Key }}
```

Comment:
Same comment here for the language key. Please change to `languageKey` for consistency.